### ISSUE TITLE
Incorrect formatting ID attribute of anchor

### ISSUE THREAD

**Issue by RyochanUedasan** [prettier/plugin-pug#63]
# 
## version
- pug-cli@1.0.0-alpha6
- prettier@1.18.2
- @prettier/plugin-pug@1.1.4

## example
- Origin ( `test.pug` )
```
a(href="/index.html")#xxx.btn
a#xxx(href="/index.html").btn
```

- Compiled (`test.html`)
```
$ ./node_modules/.bin/pug test.pug 
<a class="btn" href="/index.html" id="xxx"></a><a class="btn" id="xxx" href="/index.html"></a>
```

- Formatted
```
$ ./node_modules/.bin/prettier "test.pug"  
a(href="/index#xxx.html").btn  # <- this is wierd
a#xxx(href="/index.html").btn
```

**Comment by Shinigami92** [comment]
Yes, the bug is caused by the dot between index.html
Need to improve it

**Comment by Shinigami92** [comment]
Thank you very much! 💖 
Not only did you help fixing `id` handling, you also helped identify some other issues with `id` and `class` attributes 🚀 

I have released [v1.1.5](https://www.npmjs.com/package/@prettier/plugin-pug/v/1.1.5)
Thanks for using my plugin
